src/test/java/uk/gov/ch/developer/docs/controller/developer/SignOutControllerTest.java

```diff
@@ -0,0 +1,124 @@
+package uk.gov.ch.developer.docs.controller.developer;
+
+import static org.junit.Assert.assertFalse;
+import static org.mockito.Mockito.only;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.ch.oauth.IIdentityProvider;
+import uk.gov.companieshouse.session.Session;
+import uk.gov.companieshouse.session.SessionKeys;
+import uk.gov.companieshouse.session.handler.SessionHandler;
+import uk.gov.companieshouse.session.model.SignInInfo;
+import uk.gov.companieshouse.session.store.Store;
+
+@ExtendWith(MockitoExtension.class)
+public class SignOutControllerTest {
+
+    private static final String REDIRECT_PAGE = "/home";
+
+    @Mock
+    private IIdentityProvider identityProviders;
+    @Mock
+    private Store store;
+    @Mock
+    private HttpServletRequest httpServletRequest;
+    @Mock
+    private HttpServletResponse httpServletResponse;
+    @Mock
+    private Session session;
+
+    private final SignInInfo signInInfo = new SignInInfo();
```

Comment:
Make this local to tests to prevent state change by different tests.
In general it is very dangerous to share state variables between tests, other than Mocks and constants.

Reply:
No need, as will create new test instance for each new method by default.

Reply:
https://junit.org/junit5/docs/current/user-guide/#writing-tests-test-instance-lifecycle -> for further info